pages/about.tsx

```diff
@@ -120,6 +120,28 @@ const AboutPage = () => {
         calling contract with <OLink opcode="3D" title="RETURNDATASIZE" /> and{' '}
         <OLink opcode="3E" title="RETURNDATACOPY" />.
       </p>
+
+      <H2 className="mb-6">Gas costs</H2>
+      <p className="pb-8">
+        As an incentive to provide resources to run transactions, a fee is payed
+        to send and execute a transaction. The fee is determined by several
+        factors, including the amount of data sent or the amount of work that a
+        transaction requires. That fee is calculated through two mechanisms. The
+        first is a fixed cost that is defined by Ethereum, depending on what is
+        done. The unit for that is called gas, and reùains the same, though it
```

Comment:
maybe clarify `done`? Is it executed? or what done means?

Reply:
fix `reùains`

Reply:
"executed" is better thanks, done